OpenRA.Game/Primitives/Color.cs

```diff
@@ -117,9 +80,77 @@ public static Color FromArgb(uint argb)
 			return FromArgb((byte)(argb >> 24), (byte)(argb >> 16), (byte)(argb >> 8), (byte)argb);
 		}
 
+		static float SrgbToLinear(float c)
+		{
+			return c <= 0.04045f ? c / 12.92f : (float)Math.Pow((c + 0.055f) / 1.055f, 2.4f);
+		}
+
+		static float LinearToSrgb(float c)
+		{
+			return c < 0.0031308 ? c / 12.92f : ((float)Math.Pow(c, 5.0f / 12.0f) * 211.0f - 11.0f) / 200.0f;
```

Comment:
1. `c / 12.92f` should be `c * 12.92f`
2. `0031308` is missing a `f` qualifier
3. I'd suggest to keep the constants symmetrical to `SrgbToLinear`

For reference:
- linear to sRGB (encode): `f(x) = x < 0.0031308 ? x*12.92 : (1+.055)*x^(1/2.4)-0.055`
- sRGB to linear (decode): `f(x) = x < 0.04045   ? x/12.92 : ((x+.055)/(1+.055))^2.4`

Either the large constants or the common one (I'd argue the flavors of `.055` and `2.4` or more expressive than the integer equivalents.

Reply:
Done, also added a comments as discussed.